custom_components/rpi_gpio/cover.py

```diff
@@ -109,7 +109,7 @@ def __init__(self, hub, name, relay_port, relay_time, relay_active_low, relay_bi
         hub.add_cover(self, relay_port, relay_active_low, relay_bias, relay_drive,
                       state_port, state_bias, state_active_low)
 
-    def update(self):
+    def handle_event(self):
```

Comment:
There are two places that call `update` in cover:
https://github.com/thecode/ha-rpi_gpio/blob/1546133517dada69f459a64bb25148925fcfbae7/custom_components/rpi_gpio/cover.py#L128
https://github.com/thecode/ha-rpi_gpio/blob/1546133517dada69f459a64bb25148925fcfbae7/custom_components/rpi_gpio/cover.py#L142

I think they need to be updated to call `handle_event` also